js/components/skipControls.js

```diff
@@ -202,6 +220,20 @@ class SkipControls extends React.Component {
         onClick={this.onNextVideo}>
       </ControlButton>
     );
+
+    buttonTemplate[CONSTANTS.SKIP_CTRLS_KEYS.PLAY_PAUSE] = (
+      <ControlButton
+        {...this.props}
+        key={CONSTANTS.CONTROL_BAR_KEYS.PLAY_PAUSE}
+        className="oo-play-pause"
+        focusId={CONSTANTS.CONTROL_BAR_KEYS.PLAY_PAUSE}
+        ariaLabel={playPauseAriaLabel}
+        icon={playIcon}
+        tooltip={playBtnTooltip}
```

Comment:
Tooltips are not currently enabled for this component, so it might be better to remove this otherwise it would be the only one to show up

Reply:
Will do